ai-platform/snippets/predict-text-embeddings-preview.js

```diff
@@ -17,11 +17,14 @@
 'use strict';
 
 // [START generativeaionvertexai_sdk_embedding]
+
+// Calculate the embedding for code blocks. Using 'RETRIEVAL_DOCUMENT' for corpus.
+// Specify the task type as 'CODE_RETRIEVAL_QUERY' for query, e.g. 'Retrieve a function that adds two numbers'.
 async function main(
   project,
```

Comment:
I know we have a bunch of older samples that do this, but don't parameterize you're main function. See this [sample](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/security-center/snippets/v2/createFinding.js#L27-L29) for setting local variables with values to make the sample run, and marking `// TODO(developer)` to make it clear.